src/modules/AddOnProduct.rb

```diff
@@ -404,7 +409,27 @@ def GetBaseProductURL
     #     "username:password@ftp://www.example.org/dir/",
     #     "relurl://./Product_CD1/"
     #   ) -> "username:password@ftp://www.example.org/dir/Product_CD1/"
+    #   AddOnProduct::GetAbsoluteURL (
+    #     "username:password@ftp://www.example.org/dir/",
```

Comment:
Sorry this is actively misleading, suggesting that `repo:` cares about `base_url`, which it doesn't